requirements.txt

```diff
@@ -1,10 +1,11 @@
-ark-analysis @ git+https://github.com/angelolab/ark-analysis.git@v0.4.4
-mibi-bin-tools @ git+https://github.com/angelolab/mibi-bin-tools.git@v0.2.4
-tmi @ git+https://github.com/angelolab/tmi.git
+mibi-bin-tools @ git+https://github.com/angelolab/mibi-bin-tools.git@v0.2.5
+tmi @ git+https://github.com/angelolab/tmi.git@v0.0.1
```

Comment:
The conda file as well, right?

Reply:
Conda file needs the updated requirements as well.